### DISCUSSION TITLE
fans to max at program start

### DISCUSSION THREAD

**Discussion by Bobika2002** [Rem0o/FanControl.Releases#1271] [Ideas]
Hello, I have small problem with my fans, I have 4 fans at one connector just pairs are same so when I set curve with low rpm´s they don´t spin up after I manually set 100% then switch back to curve control they work fine. I had an idea if that would be easy and possible to spin all fans to max at startup for few seconds. Thanks

**Comment by Rem0o** [comment]
Set the start % to 100 % on your control cards?

**Reply by Rem0o** [reply]
![image](https://user-images.githubusercontent.com/14118956/201355263-5ce8ba5a-c129-4cb8-a17e-120179a1fc71.png)

**Comment by Bobika2002** [comment]
Okay, this fixed issue. Thanks I thought its working a bit differently! :)
